src/libraries/System.Globalization/tests/System/Globalization/TextInfoTests.cs

```diff
@@ -430,7 +436,14 @@ private static void TestToUpper(string name, string str, string expected)
         [MemberData(nameof(ToUpper_TestData))]
         public void ToUpper(string name, string str, string expected)
         {
-            TestToUpper(name, str, expected);
+            try
```

Comment:
>try [](start = 12, length = 3)

same comment as the one on the other tests

Reply:
So it looks like this situation is happening with a lot of the test data (e.g. [`CultureInfo_TestData`](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Globalization/tests/CultureInfo/CultureInfoAll.cs#L526)) where it contains locales that are not included in WASM. I don't think it's feasible to disable all of those tests for wasm.

Reply:
Is it the first time we enable the tests against wasm? how we used to test before?

Reply:
Before wasm treated locales with unavailable data the same way as Unix. It would just default to `en-us`. I think I am going to just write a second test just for the browser with the exception.